app/Platform/Commands/UpdateAchievementsUserIds.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Platform\Commands;
+
+use App\Models\Achievement;
+use App\Models\User;
+use Illuminate\Console\Command;
+use Illuminate\Support\Collection;
+
+class UpdateAchievementsUserIds extends Command
+{
+    protected $signature = 'ra:platform:achievement:update-user-ids
+                            {achievementId?}';
+    protected $description = 'Update user IDs in the Achievements table. This is a precursor to dropping the User column from the table.';
+
+    public function __construct()
+    {
+        parent::__construct();
```

Comment:
could be omitted as it's redundant.

i think a raw query would be more efficient in this case than iterating over entries in code. same result though so not too important imo.

Reply:
Good call. Command removed and SQL query added to the OP of this PR.